client/tokenized-express-checkout/index.js

```diff
@@ -324,24 +342,21 @@ jQuery( ( $ ) => {
 			} );
 
 			eceButton.on( 'shippingaddresschange', async ( event ) =>
-				shippingAddressChangeHandler( api, event, elements )
```

Comment:
The `api` argument is no longer needed when selecting a shipping address or selecting a shipping rate.